bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -389,8 +389,11 @@ Intercept of free living Nitrogen fixation with zero annual ET
 </entry>
 
 <entry id="nfix_method" type="char*25" category="cnfun_inparm"
-       group="cnfun_inparm" valid_values="Houlton,Bytnerowicz" value="Houlton" >
+       group="cnfun_inparm" valid_values="Houlton,Bytnerowicz" value="Bytnerowicz" >
 Choice of nfix parameterization
+Defaults
```

Comment:
I suggest removing this section on the defaults -- since the true defaults are in a different file. We don't do defaults elsewhere this way, and if we did it would mean having to change it in two places. So better to just have it in the one.

Reply:
I do wonder about expanding the definition above to spell out what it means more.

Maybe at least:

Choice of type of Nitrogen Fixation parameterization